.travis.yml

```diff
@@ -30,8 +42,9 @@ matrix:
             - gcc-5
             - g++-5
       env: DEVITO_ARCH=gcc-5 DEVITO_OPENMP=0 RUN_EXAMPLES=False INSTALL_TYPE=conda MPI_INSTALL=0
-    - os: linux
-      python: "3.6"
+    - python: 3.7
+      dist: xenial
```

Comment:
why fixing xenial?

Reply:
3.7 only supported on xenial dist, so has to be put there.